201-vm-domain-join/azuredeploy.json

```diff
@@ -14,15 +14,15 @@
         "description": "Existing subnet that contains the domain controller"
       }
     },
-    "dnsLabelPrefix": {
+    "vmName": {
```

Comment:
Change made and committed. https://github.com/Azure/azure-quickstart-templates/pull/1552/commits/ff19d0db333c5ff4bea1f3f418aec7a1f0d9ff78